dydx/client.py

```diff
@@ -72,96 +53,187 @@ def _delete(self, *args, **kwargs):
 
     # ------------ Public API ------------
 
-    def get_pairs(self):
+    def get_pairs(
+        self
+    ):
         '''
         Return all tradable pairs
+
         :returns: Array of trading pairs
-        :raises: Exception
+
+        :raises: DydxAPIError
         '''
-        return self._get('pairs')
+        return self._get('dex/pairs')
 
-    def get_balances(self, **params):
+    def get_balances(
+        self,
+        address=None,
+        number=None
+    ):
         '''
         Return all balances for an address and account number
-        :param owner: required
-        :type owner: string
-        :param account_number: required
-        :type account_number: string
+
+        :param address: optional
+        :type address: address string
+
+        :param number: optional
+        :type number: number
+
         :returns: Array of balances
-        :raises: Exception
-        '''
-        return self._get('balances', params=params)
 
-    def get_orders(self, **params):
+        :raises: DydxAPIError
+        '''
+        if address is None:
+            address = self.public_address
+        if number is None:
+            number = self.account_number
+
+        return self._get('accounts/' + address, params=utils.remove_nones({
+            'number': number
+        }))
+
+    def get_orders(
+        self,
+        makerAccountOwner=None,
+        makerAccountNumber=None,
```

Comment:
Same comment here about this account number.

Reply:
should be good!